aws/infrastructure.tf

```diff
@@ -95,6 +133,13 @@ resource "aws_instance" "instances" {
   }
 
   depends_on = [aws_internet_gateway.gw]
+
+  # spot specific variables
+  wait_for_fulfillment   = lookup(each.value, "wait_for_fulfillment", true)
+  spot_type              = lookup(each.value, "spot_type", "one-time")
```

Comment:
Wouldn't you want this to be `persistent` so that if you lose a node it gets replaced? Or would that not really be supported?

Reply:
If the node is destroyed, it could be replaced without issue, so yeah `persistent` might be better.